tsconfig.json

```diff
@@ -0,0 +1,13 @@
+{
+	"compilerOptions": {
+		"target": "es6",
+		"module": "esnext",
+		"moduleResolution": "node",
+		"jsx": "react-jsx",
+		"strict": true,
+		"esModuleInterop": true,
+		"skipLibCheck": true,
+		"forceConsistentCasingInFileNames": true
+	},
+	"include": ["src"]
+}
```

Comment:
The build works on my end now with the vite command. 

I wonder if it was the file name change from vit2.config.js > vite.config.ts that did it.

I also see the built files in `/build` too which makes things nicer. 🙌🏻